src/sql/store.py

```diff
@@ -77,8 +77,10 @@ def store(self, key, query, with_=None):
             )
         if with_ and key in with_:
             raise ValueError(f"Script name ({key!r}) cannot appear in with_ argument")
+        # Remove trailing semicolon from query
+        query_without_semicolon = query.replace(";", "")
```

Comment:
this won't work. the query might contain non-trailing `;` characters that are required to run the query.

Reply:
yeah, so in any table that has string columns, you might want to filter the values. example, in duckdb: https://duckdb.org/docs/sql/functions/patternmatching.html

```sql
select *
from table
where some_col LIKE `%;%`
```

this query should return rows whose some_col contain `;`

Reply:
but since we're replacing `;`, with ``, the user's query will be transformed into:

```
select *
from table
where some_col LIKE `%%`
```

Reply:
got it, pushed up a new commit with a new unit test as well to cover that case. that should fix it!